.github/workflows/tests_linux.yml

```diff
@@ -486,4 +486,58 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           fail_ci_if_error: true
-          token: ${{ secrets.CODECOV_TOKEN }}
\ No newline at end of file
+          token: ${{ secrets.CODECOV_TOKEN }}
+
+  cpptestsWithMultipleBackends:
+    strategy:
+      matrix:
+        os: [ubuntu-22.04]
```

Comment:
```suggestion
        os: [ubuntu-22.04]
        exec_model: [\"SERIAL\"]
        kokkos_version: ${{ fromJson(needs.build_and_cache_Kokkos.outputs.kokkos_version) }}
```

Reply:
:rofl: one more patch... to use the cached kokkos!